in_toto/model.go

```diff
@@ -79,6 +79,8 @@ const (
 	// The SPDX mandates 'spdxVersion' field, so predicate type can omit
 	// version.
 	PredicateSPDX = "https://spdx.dev/Document"
+	// PredicateCycloneDX represents a CycloneDX SBOM
+	PredicateCycloneDX = "https://cyclonedx.org/schema"
```

Comment:
Reason for `/schema` vs. `/Document` (as in SPDX version)?

Reply:
I think Document in spdx tries to refer to a SPDX document somehow. I pointed the URL to where the CycloneDX schemas live to make it more in line with the other predicate URLs.

Reply:
LGTM